samples/samples-js/AddProductsArray/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const products = [{
+        "productid": 1,
+        "name": "Cup",
+        "cost": 2
+    }, {
+        "productid": 2,
+        "name": "Glasses",
+        "cost": 12
+    }];
+
+    context.bindings.products = req["body"] ?? JSON.stringify(products);
```

Comment:
Let's keep this the same as the C# function and only insert the local items directly, not parse from body. 

(Also you should generally access properties with . notation, e.g. `req.body`)